src/components/policy/policy_regular/include/policy/policy_manager.h

```diff
@@ -1,486 +0,0 @@
-      const std::string& priority) const = 0;
-
-  /**
-   * @brief Allows to update Vehicle Identification Number in policy table.
-   * @param new value for the parameter.
-   */
-  virtual void SetVINValue(const std::string& value) = 0;
-
-  /**
-   * @brief Checks, if application has policy assigned w/o data consent
-   * @param policy_app_id Unique application id
-   * @return true, if policy assigned w/o data consent, otherwise -false
-   */
-  virtual bool IsPredataPolicy(const std::string& policy_app_id) const = 0;
-
-  /**
-   * Returns heart beat timeout
-   * @param app_id application id
-   * @return if timeout was set then value in milliseconds greater zero
-   * otherwise heart beat for specific application isn't set
-   */
-  virtual uint32_t HeartBeatTimeout(const std::string& app_id) const = 0;
-
-  /**
-   * @brief SaveUpdateStatusRequired alows to save update status.
-   */
-  virtual void SaveUpdateStatusRequired(bool is_update_needed) = 0;
-
-  /**
-   * @brief Handler on applications search started
-   */
-  virtual void OnAppsSearchStarted() = 0;
-
-  /**
-   * @brief Handler on applications search completed
-   */
-  virtual void OnAppsSearchCompleted() = 0;
-  /**
-   * @brief Gets request types for application
-   * @param policy_app_id Unique application id
-   * @return request_types Request types of application
-   */
-  virtual const std::vector<std::string> GetAppRequestTypes(
-      const std::string policy_app_id) const = 0;
-
-  /**
-   * @brief Get information about vehicle
-   */
-  virtual const VehicleInfo GetVehicleInfo() const = 0;
-
-  /**
-   * @brief OnAppRegisteredOnMobile alows to handle event when application were
-   * succesfully registered on mobile device.
-   * It will send OnAppPermissionSend notification and will try to start PTU.
-   *
-   * @param application_id registered application.
-   */
-  virtual void OnAppRegisteredOnMobile(const std::string& application_id) = 0;
-
-  /**
-   * @brief RetrieveCertificate Allows to obtain certificate in order
-   * to start secure connection.
-   *
-   * @return The certificate in PKCS#7 format.
-   */
-  virtual std::string RetrieveCertificate() const = 0;
-
-  virtual const PolicySettings& get_settings() const = 0;
-
- protected:
-  /**
-   * Checks is PT exceeded IgnitionCycles
-   * @return true if exceeded
-   */
-  virtual bool ExceededIgnitionCycles() = 0;
-
-  /**
-   * Checks is PT exceeded days
-   * @return true if exceeded
-   */
-  virtual bool ExceededDays() = 0;
-
-  /**
-   * @brief StartPTExchange allows to start PTU. The function will check
-   * if one is required and starts the update flow in only case when previous
-   * condition is true.
-   */
-  virtual void StartPTExchange() = 0;
-};
-
-}  // namespace policy
-
-extern "C" policy::PolicyManager* CreateManager();
-extern "C" void DeleteManager(policy::PolicyManager*);
-
-#endif  // SRC_COMPONENTS_POLICY_POLICY_REGULAR_INCLUDE_POLICY_POLICY_MANAGER_H_
```

Comment:
@jacobkeeler Yes, this is was just unused duplicate of src/components/include/policy/policy_regular/policy/policy_manager.h , but in scope of this PR, I added two new functions to the policy_manager interface, so if I don't delete this file, I will have to add this functions to this file too. And I think that it will be redundant.